zeg/tests/test_config.py

```diff
@@ -216,8 +305,8 @@ def test_image_config(self):
         configuration = self._get_configuration(config_data)
 
         try:
-            config.validate_config(configuration)
-        except exceptions.ValidationError:
+            config.validate_config(configuration, logger)
+        except SystemExit:
```

Comment:
This shouldn't change.

Reply:
`validate_config` will never throw a `jsonschema.exceptions.ValidatorError` because its caught and a new error raised.

Reply:
This has now been reverted due to status code return moving into main.